1-org/envs/shared/iam.tf

```diff
@@ -87,3 +87,116 @@ resource "google_organization_iam_member" "billing_viewer" {
   role   = "roles/billing.viewer"
   member = "group:${var.billing_data_users}"
 }
+
+/******************************************
+ Groups permissions according to SFB - IAM
+*****************************************/
+
+resource "google_organization_iam_member" "organization_viewer" {
+  count  = var.gcp_platform_viewer != null && var.parent_folder == "" ? 1 : 0
+  org_id = var.org_id
+  role   = "roles/viewer"
+  member = "group:${var.gcp_platform_viewer}"
+}
+
+resource "google_folder_iam_member" "organization_viewer" {
+  count  = var.gcp_platform_viewer != null && var.parent_folder != "" ? 1 : 0
+  folder = "folders/${var.parent_folder}"
+  role   = "roles/viewer"
+  member = "group:${var.gcp_platform_viewer}"
+}
```

Comment:
We should consider refactoring this into a module which takes in `parent_folder` and makes the switch btw roles. This will be more DRY, but can do that in a follow up PR too.